python/housinginsights/sources/cama.py

```diff
@@ -0,0 +1,208 @@
+                if row['properties']['NUM_UNITS']: num_units = row['properties']['NUM_UNITS']
+                if num_units == 0:
+                    num_units = 1
+
+                bedrm = row['properties']['BEDRM']
+                if bedrm == 0: bedrm = 1
+                if bedrm == None: bedrm = 0
+
+                for zone in zone_types:
+                    if zone == 'anc': zone_count = anc_count
+                    elif zone == 'census_tract': zone_count = census_count
+                    elif zone == 'neighborhood_cluster': zone_count = cluster_count
+                    elif zone == 'ward': zone_count = ward_count
+                    elif zone == 'zip': zone_count = zipcode_count
+
+                    if 'Warning' not in mar_return.keys():
+                        flag = False
+                        for dictionary in zone_count: #dictionary is {'zone_type': 'ANC', 'zone': 'ANC 8A', etc.}
+                            if dictionary['zone'] == mar_return[zone]: #mar_return[ANC] is 'ANC 8A'
+                                dictionary['housing_unit_count'] += num_units
+                                dictionary['bedroom_unit_count'] += bedrm
+                                flag = True
+                                break
+                        if not flag:
+                            zone_count.append( OrderedDict([('zone_type', zone), ('zone', mar_return[zone]), ('housing_unit_count', num_units), ('bedroom_unit_count', bedrm)]) )
+
+            except Exception as e:
+                exc_type, exc_obj, exc_tb = sys.exc_info()
+                print(exc_type, "line", exc_tb.tb_lineno)
+                print("Error! SSL: ", row['properties']['SSL'], row['properties']['AYB'])
+                continue
+
+        return {'anc': anc_count, 'census_tract': census_count, 'neighborhood_cluster': cluster_count, 'ward': ward_count, 'zip': zipcode_count}
+
+
+    def get_csv(self):
+        """
+        Takes the returned dictionary from get_data() and convert the information
+        into csv file and then save the csv file in
+        housing-insights/data/processed/zoneUnitCount
+        as zoneUnitCount_2017-05-30.csv.
+        """
+
+        if not os.path.exists('../../../data/processed/zoneUnitCount'):
+            os.makedirs('../../../data/processed/zoneUnitCount')
+
+
+        data_processed_zoneUnitCount = os.path.join(PYTHON_PATH, os.pardir, 'data', 'processed', 'zoneUnitCount')
+
+        zone_data = self.get_data()
+        toCSV = []
+        date = datetime.date.today().strftime('%Y-%m-%d')
+        filename = os.path.join(data_processed_zoneUnitCount, 'zoneUnitCount_'+date+'.csv')
+
+        for key, value in zone_data.items():
+            toCSV.extend(value)
+
+        keys = toCSV[0].keys()
+
+        with open(filename, 'w') as output_file:
+            dict_writer = csv.DictWriter(output_file, keys)
+            dict_writer.writeheader()
+            dict_writer.writerows(toCSV)
+
+
+my_api = CamaApiConn()
```

Comment:
To make this able to be imported into /python/scripts/load_data.py, all you have to do is add an `if __name__ == '__main__':` statement just above this line. that way it won't automatically run when it is imported, instead it will make the objects available; but it will run if you run this specific file from the command line.